app/Phragile/RenderedTask.php

```diff
@@ -0,0 +1,119 @@
+<?php
+namespace Phragile;
+
+class RenderedTask {
```

Comment:
True. So what I did now is introducing Domain and Presentation namespaces and having two Task classes in each namespace.